packages/jaspr/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+- Added `onClick` override to `a` html tag. Fixing issues with default action handling. @slightfoot
```

Comment:
I never add contributor tags to the changelog, I feel it would be unfair to other contributors to start now.